File: program/find-minimum-of-n-numbers/find_minimum_of_n_numbers.rs
fn find_minimum_of_n_numbers(nums: &mut [i32]) -> i32 {
    nums.sort();
    return nums[0];
}

// testing
fn main() {
    let mut arr: [i32; 4] = [1, 5, 3, 6];
    let rs = find_minimum_of_n_numbers(&mut arr);
    println!("Minimum of n numbers is {:?}", rs);
}

File: program/find-minimum-of-n-numbers/FindMinimumOfNNumbers.java
import java.util.*;

public class FindMinimumOfNNumbers {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    System.out.print("Enter number of elements in the array:");
    int n = sc.nextInt();
    int[] arr = new int[n];

    System.out.println("Enter elements of array:");
    for (int i = 0; i < n; i++) {
      arr[i] = sc.nextInt();
    }

    int mini = Integer.MAX_VALUE;
    for (int i = 0; i < n; i++) {
      if (arr[i] < mini) mini = arr[i];
    }

    System.out.println("Minimum value: " + mini);
  }
}
